### ISSUE TITLE
File unincluded

### ISSUE THREAD

**Issue by GodofHope** [typescript-eslint/typescript-eslint#897]
**What code were you trying to parse?**

.eslintrc.js:
```ts
  parserOptions: {
    parser: '@typescript-eslint/parser',
    project: './tsconfig.json'
  },
```
tsconfig.json:
```
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "strict": true,
    "jsx": "preserve",
    "importHelpers": true,
    "moduleResolution": "node",
    "experimentalDecorators": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "sourceMap": false,
    "baseUrl": ".",
    "types": [
      "webpack-env",
      "jquery"
    ],
    "lib": [
      "esnext",
      "dom",
      "dom.iterable",
      "scripthost"
    ]
  },
  "include": [
    "src/**/*.ts",
    "src/**/*.tsx",
    "src/**/*.vue"
  ],
  "exclude": [
    "dist",
    "node_modules"
  ]
}
```

**What did you expect to happen?**

No error. Vue file has been included in the tsconfig.json.

**What actually happened?**

 Parsing error: If "parserOptions.project" has been set for @typescript-eslint/parser, U:\website\nenuyouth\src\views\tools\NetCharge.vue must be included in at least one of the projects provided at src\views\tools\NetCharge.vue:

Parsing error: If "parserOptions.project" has been set for @typescript-eslint/parser, U:\website\nenuyouth\src\views\tools\SchoolGzh.vue must be included in at least one of the projects provided at src\views\tools\SchoolGzh.vue

and so on.

**Versions**

| package                     | version |
| --------------------------- | ------- |
| `@typescript-eslint/parser` | `2.0.0` |
| `TypeScript`                | `3.5.3` |
| `ESLint`                    | `5.16.0` |
| `node`                      | `10.16.3` |
| `npm`                       | `6.9.0` |

**Comment by bradzacher** [comment]
https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/parser#configuration

see `extraFileExtensions`

**Comment by GodofHope** [comment]
Thanks, my mistake.
